core/src/main/java/com/xmartlabs/base/core/module/GsonModule.java

```diff
@@ -27,52 +14,26 @@
 
 @Module
 public class GsonModule {
-  static final String SERVICE_GSON_NAME = "ServiceGson";
+  public static final String SERVICE_GSON_NAME = "ServiceGson";
```

Comment:
should it be in the core? In my opinion the project should have multiple gson modules. This module should provide only the common gson object